docs/naps/6-contributable-menus.md

```diff
@@ -153,34 +153,43 @@ actions.
-top level menu therefore contains five submenus organized by the types of processing
+top level menu therefore contains a number of submenus organized by the types of processing
 the user may wish to perform on the selected `Layer` or `Layers`.
 
 The `Layers` submenus are organized to give the user an immediate
 feeling of what might happen to their `Layers` as a result of clicking
 one of these menu items.
 
-1. `Visualization` - Items in this submenu allow you to generate visualizations from selected layer or layers.
-They do not change the layer data.
-2. `Measure` - Items in this submenu provide utilities for summarising information about your layer's data.
-3. `Edit` - The items in this submenu change the data of your layer through `Filters` or `Transformations`.
-Additionally `Annotation Tools` provide a location for convenience layer editing tools e.g. `Labels` split/merge actions.
-Items in this submenu **should not** generate new layers, but rather act upon the existing layer data.
-3. `Generate` - Items in this submenu are the main *analysis* actions you can take on your layer.
-These items should add new layers to the viewer based on analyses and processing of data in your selected layer(s).
-The five proposed submenus are `Projection`, `Segmentation`, `Classification`, `Registration` and `Tracks`.
+1. `Visualize` - Items in this submenu allow you to change visualization parameters from selected layer or layers.
+They do not change the layer data, but will change how it's displayed. An example would be a widget that allows you
+to select which subset of points is visible based on feature properties.
+2. `Measure` - Items in this submenu provide utilities for summarising information about your layer's data. An example
+would be a widget that plots the change in the intensity of a clicked pixel over time.
+3. `Edit` - The items in this submenu allow you to change the data of your layer through `Filter` actions e.g. a gaussian 
+filter or `Transform` actions e.g. an affine transform. Additionally `Annotate` provides a location for convenient layer 
+editing tools e.g. `Labels` split/merge actions. Items in this submenu **should not** generate new layers, but rather act upon 
+the existing layer data.
+
+The remaining submenus under `Layers` deal with generating new layers based on processing of existing layers
+or other information. They define common *analysis* actions users may want to take on their layers.
+
+4. `Register` - Commands that allow the user to perform image registration on one or more layers
+5. `Project` - Commands that generate various projections based on one or more layers
+6. `Segment` - Commands that generate instance segmentations e.g. watershed, SLIC, threshold
+7. `Track` - Commands that take in an image or segmentation and produce timelapse tracks
+8. `Classify` - Commands that predict classes/categories for pixels, objects, points, etc.
```

Comment:
Question, would these be submenus in `Layers` or would they one layer deeper e.g., `Layers/Processing/...` ? (just thinking about how long the `Layers` menu would be)